include/zephyr/net/wifi.h

```diff
@@ -38,6 +38,18 @@
 extern "C" {
 #endif
 
+/** @brief IEEE 802.11 SUITE-B */
+enum wifi_cipher_suite_b {
```

Comment:
Could you make it align with 'enum wifi_suiteb_type', then remove one of those to avoid duplicate implementation.